tasks/build-i18n.js

```diff
@@ -0,0 +1,59 @@
+// Contains Grunt tasks related to building translatable assets.
+module.exports = function(grunt) {
+  var path = require('path');
+  var MessageFormat = require('messageformat');
+
+  grunt.registerMultiTask('build-i18n', 'Compile i18n strings!', function() {
+    this.files.forEach(function(file) {
+      file.src.forEach(function(src) {
+        var locale = path.basename(src, '.json').toLowerCase();
+        var englishData = grunt.file.readJSON(src.replace(locale, 'en_us'));
+        var localeData = grunt.file.readJSON(src);
+        // Remove any strings which are empty
+        Object.keys(localeData).forEach(function(key) {
+          if (localeData[key] === '') {
+            delete localeData[key];
+          }
+        });
+        // Fill in missing strings with the English strings
+        var finalData = Object.assign(englishData, localeData);
```

Comment:
I noticed that sometimes there is a var and other times a let. Do we prefer one or the other? 

There is a let on line 25

Reply:
oops that should be `var` then. I noticed `var` is mostly used in this project.